src/components/input.js

```diff
@@ -12,7 +12,7 @@ import * as Utils from 'src/libs/utils'
 const styles = {
   input: {
     height: '2.25rem',
-    border: `1px solid ${colors.light()}`, borderRadius: 4
+    border: `1px solid ${colors.dark(0.55)}`, borderRadius: 4
```

Comment:
Got approval from Lou on this change - makes it consistent between selects and inputs now to both have dark borders